OSDOCS-5277-rn-update: Update the CSI ref in a vSphere RN (#58180)
openshift/openshift-docs: release_notes/ocp-4-13-release-notes.adoc

[IMPORTANT]
====
The VMware vSphere region and zone enablement feature is only available with a newly installed cluster, because this type of cluster includes the Container Storage Interface (CSI) automatic migration feature. The CSI automatic migration feature sets CSI storage class as the default storage class. 

> If you want to upgrade a cluster from a previous release, you must manually enable the CSI automatic migration feature, and then ensure that CSI storage is set as the default storage class for your cluster.

Reviewer: The VMware vSphere region and zone enablement feature is only available with a newly installed cluster,  because this requires vSphere CSI driver as default storage driver in the cluster. 

A cluster that was upgraded from previous release defaults to intree vSphere driver and hence CSI automatic migration must be enabled on such clusters before multiple zones and regions can be configured in upgraded clusters.

[IMPORTANT]
====
The VMware vSphere region and zone enablement feature is only available with a newly installed cluster, because this feature requires the vSphere Container Storage Interface (CSI) driver as the default storage driver in the cluster.

> A cluster that was upgraded from a previous release defaults to using the in-tree vSphere driver, so you must enable CSI automatic migration for the cluster. You can then configure multiple regions and zones for the upgraded cluster. 

Author: Hi @kcarmichael08 . I will be adding links after all my feature content is merged; that way, I can test the links. The CSI automatic migration is actively been discussed among engineers and content strategists, so I do not have a link for this section just yet as the planning stage of the writing is still in progress on another team (storage).

> A cluster that was upgraded from a previous release defaults to using the in-tree vSphere driver, so you must enable CSI automatic migration for the cluster. You can then configure multiple regions and zones for the upgraded cluster. 

Reviewer 2: Sounds good! :)

> The VMware vSphere region and zone enablement feature is only available with a newly installed cluster, because this feature requires the vSphere Container Storage Interface (CSI) driver as the default storage driver in the cluster.
> 
> A cluster that was upgraded from a previous release defaults to using the in-tree vSphere driver, so you must enable CSI automatic migration for the cluster. You can then configure multiple regions and zones for the upgraded cluster. 

Reviewer 3: ```suggestion
The VMware vSphere region and zone enablement feature is only available with a newly installed cluster because it requires the vSphere Container Storage Interface (CSI) driver as the default storage driver in the cluster.

A cluster that was upgraded from a previous release defaults to the in-tree vSphere driver. As a result, you must enable CSI automatic migration for the cluster to use the feature. You can then configure multiple regions and zones for the upgraded cluster. 
```

Suggestion.

Author: "It" might be misconstrued here, as it can be a vague referent: the cluster or the feature. I do not want our customers thinking that the CSI driver being set as the default storage driver is essential for all newly installed cluster on vSphere.

Author: How about:

> TO use this feature on a newly installed cluster, you must enable CSI automatic migration for the cluster.

Reviewer 3: Fine by me.
